modules/common/ca-issuer/main.tf

```diff
@@ -46,6 +46,7 @@ resource "kubernetes_secret" "ca" {
   }
 }
 
+#tfsec:ignore:general-secrets-sensitive-in-attribute
```

Comment:
this may also be an issue

Reply:
this rule is kinda annoying, because it seems like it's just going to trigger any time it sees a variable with the word "secret" in it that isn't marked as `sensitive`.  in reality, the variable it's talking about, which I assume is `ca_secret`, is actually the name of a Kubernetes secret to reference.  so it isn't itself a sensitive value.

I imagine a lot of these violations are due to us referencing the name of a Kubernetes secret that isn't supposed to be sensitive.